tasks/main.yml

```diff
@@ -9,6 +9,7 @@
     environment: "{{ cluster_environment }}"
     template: "{{ cluster_template }}"
     parameters: "{{ cluster_parameters }}"
+    wait: "{{ cluster_stack_update_only }}"
```

Comment:
This logic is the wrong way round - this will only wait when `cluster_stack_update_only=true` which is the opposite of what you are trying to achieve.